hack/release-imp.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+#
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -ex
+
+TAG_NAME=$1
+DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" > /dev/null && pwd)"
+
+if [[ -z "$TAG_NAME" ]]; then
+  echo "Usage: release.sh <release-tag>" >&2
+  exit 1
+fi
+
+"$DIR/../components/release-in-place.sh" $TAG_NAME
+"$DIR/../manifests/gcp_marketplace/hack/release.sh" $TAG_NAME
+"$DIR/../manifests/kustomize/hack/release.sh" $TAG_NAME
+"$DIR/../sdk/hack/release.sh" $TAG_NAME
+echo "$TAG_NAME" > "$DIR/../VERSION"
```

Comment:
wondering whether move these release sh to tool or release folder but not the manifest folder is better.
Especially under the kustomize folder seems not look so safe to me.

Reply:
I'd prefer each folder to be self contained, so they are easily found and maintained.
The `hack` folder naming is a convention starting from kubernetes project for storing not-so-robust scripts that we need anyway: https://www.reddit.com/r/golang/comments/b95sd6/whats_the_recent_pattern_of_a_hack_directory_in/